service-map/src/main/resources/META-INF/OskariLayer.xml

```diff
@@ -616,6 +641,30 @@
 
         values (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)
     </statement>
+    
+    <update id="updateOrder" parameterClass="java.util.HashMap">
+        WITH cte AS (
+                SELECT maplayerid
+                FROM   oskari_maplayer_group_link
+                WHERE  maplayerid=#layerId#
+                       AND groupid=#groupId#
+                LIMIT  1
+                FOR    UPDATE
+        )
+        UPDATE oskari_maplayer_group_link omgl
+        SET    order_number = #orderNumber#
+        FROM   cte
```

Comment:
I'd argue the cte makes this more difficult to read. We could just replace the
```
FROM cte
WHERE oml.maplayerid = cte.maplayerid
```
with
```
WHERE maplayerid = #layerId#
AND groupid = #groupId#
```
right? That way we can also drop the table aliases as well as the whole WITH statement.